### PR TITLE
[UI-10] Past Speakers

### LINKED ISSUE #5: [UI-10] Past Speakers Section
### Requirements
 - Has animations
 - Content is only *visually* hidden
 - Use `gatsby-image`
 - Height is dynamic based on tallest card
 - 2 column (centered) in tablet, 1 column mobile

### Mockup Reference
Screen Shot 2022-04-25 at 1 34 25 AM

### PR DESCRIPTION
**Pull request by fpunny**
Closes #5 

### Responsiveness

https://user-images.githubusercontent.com/23289083/166200213-5dc40d00-7f82-4193-8704-145147b3b7e1.mov

### Interactiveness

https://user-images.githubusercontent.com/23289083/166200306-75cee50f-84ac-4df5-b80b-98a10ab4c18e.mov

### CODE REVIEW

**Review by BlazingAsher — APPROVED** [review]
LGTM

**Comment by henrytwo** [comment]
Is this broken on the dev site? (https://develop--ht6-landing-2022.netlify.app/) The pictures don't seem to be in the right place

image

```
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.127 Safari/537.36
```

**Comment by BlazingAsher** [comment]
Interesting, broken for me as well on develop. Probably some conflict that happened during merge?

**Comment by fpunny** [comment]
nani the fk. it seems that the ordering of the styles changed on build (a notorious bug in gatsby nowadays >.>)

tldr; `.gatsby-image-wrapper` is overwriting my position absolute with relative
